scripts/build_camera.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+cd ..
```

Comment:
Using this assumes that the script is run from inside the folder `scripts` which is not guaranteed.

Reply:
one way to make sure that you go into the root of the project is to use 
`cd $(dirname $(dirname $(realpath $0)))`
this assumes that the script is placed one level below the root, but it can be ran from wherever.